### ISSUE TITLE
several columns in tasklistview

### ISSUE THREAD

**Issue by oroulet** [QTodoTxt/QTodoTxt2#19]
Did we completely give up that idea? I think this would have a value for the priority, the current solution (with space) get complaints https://github.com/QTodoTxt/QTodoTxt/issues/428.
So we could have two columns, one for priority and one for the rest. the prority column could disapear or simply get reset while editing...just an idea, now that you have more experience with qml

**Comment by fuchsmich** [comment]
I agree with the complaint. I would completly remove the space before 
the tasks without priority even if there are both tasks with and without 
prio. So I also don't see much use for an extra column.

...but the decision is up to you, boss ;)


Am 12.05.2017 um 09:05 schrieb oroulet:

**Comment by oroulet** [comment]
I am not the boss ;-) I just inherited that code, there are not many lines left from what I inherited, but this is one the things. in free software those that write code decide. But it is a good idea to listen to others yes ;-)

**Comment by oroulet** [comment]
btw it looks like listview does not support multiselection while treeview and tableview do.... one more reason to switch to tableview...

**Comment by fuchsmich** [comment]
I'm playing around with tableview:
https://github.com/fuchsmich/QTodoTxt2/tree/tableview

**Comment by oroulet** [comment]
Make a PR then it is easier to see changes

**Comment by fuchsmich** [comment]
It's in a very early state, but looks promising.
